### PR TITLE
never let 'person is typing' message stay alive too long

### PR DESCRIPTION
**Pull request by Connoropolous**

### CODE REVIEW

File: src/components/MessageForm/component.js [hunk 1]
```diff
@@ -72,14 +72,9 @@ export default class MessageForm extends React.Component {
...
   // typing.
-  //
-  // then, 8 seconds after typing stops, broadcast "I'm not typing!". if typing
-  // resumes, cancel the 8-second countdown.
   startTyping = throttle(() => {
     this.sendIsTyping(true)
-    if (this.queuedStop) clearTimeout(this.queuedStop)
-    this.queuedStop = setTimeout(() => this.sendIsTyping(false), 8000)
-  }, 5000)
+  }, 3000)
```
Comment (levity) [inline src/components/MessageForm/component.js]: can you export `STARTED_TYPING_INTERVAL` from CommentForm and use it here?

**Review by levity — CHANGES_REQUESTED** [review]
Other than the duplicated constant, looks great

**Comment by Connoropolous** [comment]
Yepp!

File: src/components/PeopleTyping.js [hunk 2]
```diff
@@ -1,9 +1,12 @@
 import React from 'react'
-import { values } from 'lodash'
+import { each, keys, values } from 'lodash'
 import cx from 'classnames'
 import { getSocket } from '../client/websockets'
+import { STARTED_TYPING_INTERVAL } from './CommentForm/component'
 const { bool } = React.PropTypes
 
+const MS_CLEAR_TYPING = STARTED_TYPING_INTERVAL + 1000
```
Comment (Connoropolous) [inline src/components/PeopleTyping.js:8]: @levity that made me think to import it here too and make this clear

**Comment by Connoropolous** [comment]
should be good now

File: src/components/PeopleTyping.js (hunk 2 above)
Comment (levity) [reply src/components/PeopleTyping.js:8]: good call

**Review by levity — APPROVED** [review]

**Pull request merged**